cruise.umple/src/Parser_Code.ump

```diff
@@ -1785,6 +1860,20 @@ class ErrorTypeSingleton
     parse(new TextParser(filename, input));
   }
 
+  private static <T extends Closeable> void closeAsRequired(T reader){
```

Comment:
This is being repeated around, why are we not having this in a method. I thought I saw it in `SampleFileWriter` (a horrible name, I know).